go/filemanager/pfsmod/resp.go

```diff
@@ -0,0 +1,48 @@
+package pfsmod
+
+//JSONResponse is the common response from server
+type JSONResponse struct {
```

Comment:
The `result` field type of all response struct is `interface{}` in Go,  so maybe we can only need one struct named:
```golang
type Response struct {
    Err string `json:"err"`,
    Result interface{} `json:"result"`
}
```